pkg/operator/controller.go

```diff
@@ -382,6 +400,11 @@ func (c *Controller) Run(stopCh <-chan struct{}) error {
 		}
 	}
 
+	err = c.clusterStatus.Create(c.listers)
+	if err != nil {
+		glog.Errorf("unable to create cluster operator resource: %s", err)
+	}
```

Comment:
i know you just moved this code here, but isn't it going to report an error on every subsequent startup because the resource already exists?  we should probably check if it's an already exists error and not log that.